docs/en/users/user_queries.md

```diff
@@ -0,0 +1,63 @@
+# User queries
+
+*User queries* are a way to store any FreshRSS search query.
+
+Read about [the filters](./10_filter.md) to learn the different ways to search and filter
+articles in FreshRSS.
+
+## Bookmark the current query
+
+Once you have a search query with a filter, it can be saved.
+
+To do so, display the user queries drop-down menu by clicking the button next to the state buttons:
+
+![User queries drop-down](../img/users/user.queries.drop-down.empty.png)
+
+Then click on the bookmark action.
+
+## Using a bookmarked query
+
+Display the user queries drop-down menu by clicking the button next to the state buttons:
+
+![User queries drop-down](../img/users/user.queries.drop-down.not.empty.png)
+
+Then click on the bookmarked query, the previously stored query will be applied.
+
+> ℹ️ Note that only the search query is stored, not the articles.
+> So the results you are seeing one day might be different another day.
+
+## Share your user queries
+
+A prerequisite is that the FreshRSS API(s) must be enabled in FreshRSS authentication settings.
+
+From the configuration page of the user queries,
+it is possible to share the output of the user queries with external users,
+in the formats HTML, RSS, and OPML:
+
+![Share user query](../img/users/user-query-share.png)
+
+> ℹ️ Note that the sharing as OPML is only available for user queries based on all feeds, a category, or a feed.
+> Sharing by OPML is **not** available for queries based on user labels or favourites or important feeds,
+> to avoid leaking some feed details in an unintended manner.
+
```

Comment:
Let us think to give a note here, that this feature will be available with 1.24 (it will note, that it is not yet available)